memory_server.py

```
@@ -31,6 +31,22 @@ class HistoryRequest(BaseModel):
 settings_manager = ImportantSettingsManager()
 time_manager = TimeIndexedMemory(recent_history_manager)
 
+def reload_memory_components():
+    """重新加载记忆组件配置（用于新角色创建后）"""
```

Ruff (0.14.6)

Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)